src/game.cpp

```diff
@@ -176,13 +177,17 @@ void Game::init() {
 			throw CreateException("Quantum Break is not yet supported by OpenAWE.");
 		case kResultAlanWakeRemastered:
 			throw CreateException("Alan Wake Remastered is not yet supported by OpenAWE.");
+		case kResultControl:
+			throw CreateException("Control is not yet supported by OpenAWE.");
+			//prefix = "data/";
+			break;
 		case kResultUnknown:
 		default:
 			throw CreateException("Unknown game data was supplied.");
 	}
 
 	// Check if the resources have packmeta files and load them and if not load streamed resources
-	if (engine == kAlanWakesAmericanNightmare) {
+	if (engine == kAlanWakesAmericanNightmare || engine == kControl) {
```

Comment:
As of right now, if `engine == kControl`, an exception will be thrown before reaching the statement, making this new check redundant.